tasks/aliases.yaml

```diff
@@ -5,6 +5,7 @@ test:
   - connect:serve
   - sauce_connect
   - protractor
```

Comment:
I think protractor should be removed as grunt-protractor-runner is not installed now